Microsoft.DotNet.Interactive.PowerShell/PowerShellKernel.cs

```diff
@@ -104,6 +104,24 @@ private PowerShell CreatePowerShell()
                 PSModulePathEnvName,
                 $"{psJupyterModulePath}{Path.PathSeparator}{psModulePath}");
 
+            // Set $PROFILE.
+            string allUsersCurrentHost = DollarProfileHelper.GetFullProfileFilePath(forCurrentUser: false);
```

Comment:
I suggest to move the all these to `DollarProfileHelper` and only calls one method here to setup `$profile` variable and execute the profile.